packages/react-ai/src/components/AIConversation/views/default/Form.tsx

```diff
@@ -11,15 +11,49 @@ import { ComponentClassName } from '@aws-amplify/ui';
 import { ControlsContextProps } from '../../context/ControlsContext';
 import { Attachments } from './Attachments';
 import { LoadingContext } from '../../context/LoadingContext';
+import { ConversationInputContext } from '../../context';
 
 function isHTMLFormElement(target: EventTarget): target is HTMLFormElement {
   return 'form' in target;
 }
 
+/**
+ * Will conditionally render the DropZone if allowAttachments
+ * is true
+ */
+const FormWrapper = ({
```

Comment:
A bit confused on whether this addition is creating deltas between the behaviors of the default `AIConversation` and the Amplify UI version?

Reply:
Yes, so the Amplify UI version uses the DropZone component that wraps the form so users can drag and drop a file onto the form and it gets attached (in addition to a button which triggers the file picker). So `allowAttachments` will affect bot the default component and the Amplify UI one.

Reply:
We do not

Reply:
we do now